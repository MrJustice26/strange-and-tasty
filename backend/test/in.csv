0.3,"1 qt. strawberries"
0.3,"Fritos or Doritos"
0.3,"2 Tbsp. margarine"
0.3,"1 cup sifted cake flour"
0.3,"1 small can chicken (all white meat)"
0.3,"1 (No. 303) can fruit cocktail (juice also)"
0.3,"1 (8 oz.) and 1 (3 oz.) pkg. Philadelphia cream cheese"
0.3,"1 lb. lean"
0.3,"1 box spaghetti"
0.3,"1 lb. pork sausage"
0.3,"1 lb. pkg. carrots"
0.3,"4 c. boiling water"
0.3,"1 envelope onion or mushroom soup mix"
0.3,"3 c. chopped potatoes"
0.3,"2 eggs"
0.3,"3 Tbsp. strawberry Jell-O"
0.3,"1 qt. tomatoes"
0.3,"8 to 10 chicken pieces"
0.3,"1 can corn"
0.3,"12 medium squash"
0.3,"1 box yellow cake mix"
0.3,"3 c. flour"
0.3,"5 lb. ground beef"
0.3,"'500 g cauliflower'"
0.3,"1 (6 oz.) pkg. raspberry jello"
0.3,"1 c. sugar"
0.3,"2 limes"
0.3,"1 (8 oz.) pkg. frozen broccoli"
0.3,"'2 bananas'"
0.3,"'200 g mixed vegetables'"
0.3,"'200 g Greek yogurt'"
0.3,"'200 g lentils'"
0.3,"4 cans (10 biscuits in each) buttermilk biscuits"
0.3,"'2 slices bread'"
0.3,"'500 g cauliflower'"
0.3,"'1 diced mango'"
0.3,"'240 g sushi rice'"
0.3,"2 c. cooked"
0.3,"1 box Twinkie cakes"
0.3,"1/4 lb. butter"
0.3,"3 1/2 c. flour"
0.3,"1 lb 2 oz red lentils"
0.3,"'200 g flour'"
0.3,"'200 g zucchini'"
0.3,"'4 chicken breasts'"
0.3,"'1 pizza crust'"
0.3,"2 boxes broccoli (16 oz. each)"
0.3," 8 oz. plain yogurt"
0.3,"1 can beef consomm"
0.3,"6 pork chops"
0.3,"1 lb. hamburger"
0.3,"1 can French-style green beans"
0.3,"1 lb. ground beef"
0.3," 6 eggs"
0.3,"1 c. sharp Cheddar cheese"
0.3,"1/2 c. oil"
0.3,"1/3 c. margarine"
0.3,"1 stick margarine"
0.3,"1/2 c. butter"
0.3,"1 (3 oz.) pkg. cream cheese"
0.3,"2 c. half and half"
0.3,"vanilla wafers"
0.3,"3/4 c. butter flavored Crisco"
0.3,"2 2/3 c. cake flour"
0.3,"4 eggs (whole)"
0.3,"red potatoes"
0.3,"1 (16 oz.) can creamed corn"
0.3,"1 can water chestnuts"
0.3,"1 lb. colorful gumdrops"
0.3,"1 1/2 c. all-purpose flour"
0.3,"1/4 c. butter"
0.3,"3/4 c. flour"
0.3,"1 c. butter or margarine"
0.3,"2 c. ham"
0.3,"1/2 c. soy sauce"
0.3,"1/2 c. plus 2 tsp. butter"
0.3,"1 (3 oz.) pkg. Philadelphia Brand cream cheese"
0.3,"1 1/2 lb. ground beef"
0.3,"1/2 stick butter"
0.3,"1 lb. hamburger"
0.3,"2 c. powdered orange drink mix"
0.3,"8 oz. curly shaped pasta"
0.3,"15 frozen Tater Tots"
0.3,"1 c. mayo or 1/2 c. mayo and 1/2 c. Miracle Whip"
0.3,"3 Tbsp. butter or margarine"
0.3,"1 1/2 c. frozen leaf spinach"
0.3,"2 cans Shoe Peg corn"
0.3,"1 lb. carrots"
0.3,"1 1/4 c. oil"
0.3,"1 c. rice (medium grain)"
0.3,"2 pkg. Mexican cornbread mix"
0.3,"1 lb. beef for stew"
0.3,"1 kg beef sirloin"
0.3,"4 large potatoes"
0.3,"2 lbs pork roast"
0.3,"4 cups flour"
0.3,"4 pork or chicken cutlets"
0.3,"2 cups flour"
0.3,"1 cup flour"
0.3,"1 head of cabbage"
0.3,"1 lb. beef tenderloin"
0.3,"100 (1 lb.) can salmon"
0.3,"22 c. warm water"
0.3," Low Calorie Quencher)"
0.3,"233 pkg. frozen broccoli cuts"
0.3,"1/2 medium head cabbage"
0.3,"2 c. sifted all-purpose flour"
0.3,"peel of 1/1 lemon"
0.3,"2 1/4 tbsp. unflavored gelatin"
0.3,"1/2 navel orange"
0.3,"1 stack of salty tears"
0.3,"1 spaghetti strand for each death"
0.3,"1 Baron Nashor"
0.3,"1 random champion youâ€™ve never played"
0.3,"1 poorly timed Flash"
0.3,"3 taco shells"
0.3,"1 teamfight victory"
0.3,"1 serving of passive-aggressive chat"
0.3,"1 enemy ADC with 20 kills"
0.3,"1 blender of tilted vibes"