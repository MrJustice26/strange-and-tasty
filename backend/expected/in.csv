0.3,"taco shells, spaghetti sauce, spaghetti, cheese, beef"
0.3,"vanilla ice cream, strawberry sauce, chocolate"
0.3,"Przepis ktorego nie znajdziesz"
0.3,"jalapeno, cheese, tomatto sauce, pizza crust"
0.3,"popcorn kernels, chili powder, melter butter"
0.3,"50 peaches, 900g fresh basil leaves, 150l balsamic vinegar 150kg honey"
0.3,"salmon, can cream of celery soup, bread crumbs, beaten eggs"
0.3,"10 sticks butter, 5 c. sugar, 15 eggs, 1l milk, crushed vanilla wafers"
0.3,"5 eggs, 1 tsp. soda, 10 coconut, 5 apples, 100ml oil, cinnamon, sugar, pecans, flour, salt, cheese"
0.3,"ground beef, parmesan, 10 cloves of chopped garlic, 10ml basil, bread crumbs, 10 eggs, olive oil, cheese, spaghetti sauce"
0.3,"500ml heavy cream, 200 ml pickle juice, 100g sugar, vanilla"
0.3,"test"
0.3,"cheese, ham, bread"